x-pack/plugins/upgrade_assistant/public/application/app.tsx

```diff
@@ -52,94 +48,71 @@ const AppHandlingClusterUpgradeState: React.FunctionComponent = () => {
 
   if (missingManageSpacesPrivilege) {
     return (
-      <EuiPageContent
-        verticalPosition="center"
-        horizontalPosition="center"
-        color="subdued"
-        data-test-subj="missingKibanaPrivilegesMessage"
-      >
-        <NotAuthorizedSection
-          title={
-            <FormattedMessage
-              id="xpack.upgradeAssistant.app.deniedPrivilegeTitle"
-              defaultMessage="Kibana admin role required"
-            />
-          }
-          message={
-            <FormattedMessage
-              id="xpack.upgradeAssistant.app.deniedPrivilegeDescription"
-              defaultMessage="To use Upgrade Assistant and resolve deprecation issues, you must have access to manage all Kibana spaces."
-            />
-          }
-        />
-      </EuiPageContent>
+      <NotAuthorizedSection
+        title={
+          <FormattedMessage
+            id="xpack.upgradeAssistant.app.deniedPrivilegeTitle"
+            defaultMessage="Kibana admin role required"
+          />
+        }
+        message={
+          <FormattedMessage
+            id="xpack.upgradeAssistant.app.deniedPrivilegeDescription"
+            defaultMessage="To use Upgrade Assistant and resolve deprecation issues, you must have access to manage all Kibana spaces."
+          />
+        }
+      />
     );
   }
 
   if (clusterUpgradeState === 'isUpgrading') {
     return (
-      <EuiPageContent
-        hasShadow={false}
-        paddingSize="none"
-        verticalPosition="center"
-        horizontalPosition="center"
-        data-test-subj="isUpgradingMessage"
-      >
-        <EuiEmptyPrompt
-          iconType="logoElasticsearch"
-          title={
-            <h1>
-              <FormattedMessage
-                id="xpack.upgradeAssistant.upgradingTitle"
-                defaultMessage="Your cluster is upgrading"
-              />
-            </h1>
-          }
-          body={
-            <p>
-              <FormattedMessage
-                id="xpack.upgradeAssistant.upgradingDescription"
-                defaultMessage="One or more Elasticsearch nodes have a newer version of
+      <EuiPageTemplate.EmptyPrompt
+        iconType="logoElasticsearch"
+        title={
+          <h1>
+            <FormattedMessage
+              id="xpack.upgradeAssistant.upgradingTitle"
+              defaultMessage="Your cluster is upgrading"
+            />
+          </h1>
+        }
+        body={
+          <p>
+            <FormattedMessage
+              id="xpack.upgradeAssistant.upgradingDescription"
+              defaultMessage="One or more Elasticsearch nodes have a newer version of
                 Elasticsearch than Kibana. Once all your nodes are upgraded, upgrade Kibana."
-              />
-            </p>
-          }
-          data-test-subj="emptyPrompt"
-        />
-      </EuiPageContent>
+            />
+          </p>
+        }
+        data-test-subj="emptyPrompt"
```

Comment:
We should preserve the same top-level `data-test-subj` so no changes are required in the test

```suggestion
        data-test-subj="isUpgradingMessage"
```